post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-z-kvm.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+:context: creating-multi-arch-compute-nodes-ibm-z-kvm
+[id="creating-multi-arch-compute-nodes-ibm-z-kvm"]
+= Creating a cluster with multi-architecture compute machine on {ibmzProductName} and {linuxoneProductName} with {op-system-base} KVM
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+To create a cluster with multi-architecture compute machines on {ibmzProductName} and {linuxoneProductName}(`s390x`) with {op-system-base} KVM, you must have an existing single-architecture bare metal (`x86_64`) cluster. For more information on bare metal installations, see xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user provisioned cluster on bare metal]. You can then add `s390x` compute machines to your {product-title} cluster on bare metal.
```

Comment:
You can then add `s390x` compute machines to your {product-title} cluster on bare metal.
-->
You can then add `s390x` compute machines to your {product-title} cluster.